app/src/store/views/orderFormView.ts

```diff
@@ -93,9 +94,9 @@ export default class OrderFormView {
   /** the markets currently open or accepting orders */
   get marketsAcceptingOrders() {
     const { MARKET_OPEN, ACCEPTING_ORDERS } = DurationBucketState;
-    return entries(this._store.batchStore.leaseDurations)
-      .map(([duration, state]) => ({ duration, state }))
-      .filter(({ state }) => state === MARKET_OPEN || state === ACCEPTING_ORDERS);
+    return this._store.batchStore.sortedDurations.filter(
```

Comment:
nit: The import `entries` is now unused (linter is complaining).

Reply:
Ah, good catch! Fixed in https://github.com/lightninglabs/lightning-terminal/pull/298/commits/e52a2cb67bd878f7f51cf52456044a5795a85fda